is_nan.json

```diff
@@ -1,56 +1,56 @@
-{
-    "id": "is_nan",
-    "summary": "Value is not a number",
-    "description": "Checks whether the specified value `x` is *not* a number. Numbers are all integers and floating-point numbers, except for the special value `NaN` as defined by the [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935).",
-    "categories": [
-        "comparison",
-        "math > constants"
-    ],
-    "parameters": [
-        {
-            "name": "x",
-            "description": "The data to check.",
-            "schema": {
-                "description": "Any data type is allowed."
-            }
-        }
-    ],
-    "returns": {
-        "description": "Returns `true` for `NaN` and all non-numeric data types, otherwise returns `false`.",
-        "schema": {
-            "type": "boolean"
-        }
-    },
-    "examples": [
-        {
-            "arguments": {
-                "x": 1
-            },
-            "returns": false
-        },
-        {
-            "arguments": {
-                "x": "Test"
-            },
-            "returns": true
-        },
-        {
-            "arguments": {
-                "x": null
-            },
-            "returns": true
-        }
-    ],
-    "links": [
-        {
-            "rel": "about",
-            "href": "https://ieeexplore.ieee.org/document/4610935",
-            "title": "IEEE Standard 754-2008 for Floating-Point Arithmetic"
-        },
-        {
-            "rel": "about",
-            "href": "http://mathworld.wolfram.com/NaN.html",
-            "title": "NaN explained by Wolfram MathWorld"
-        }
-    ]
-}
+{
+    "id": "is_nan",
+    "summary": "Value is not a number",
+    "description": "Checks whether the specified value `x` is *not* a number (NaN). Returns `true` for the special value `NaN` as defined by the [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935) and `false` otherwise.",
+    "categories": [
+        "comparison",
+        "math > constants"
+    ],
+    "parameters": [
+        {
+            "name": "x",
+            "description": "The data to check.",
+            "schema": {
+                "description": "Any data type is allowed."
+            }
+        }
+    ],
+    "returns": {
+        "description": "Returns `true` for `NaN` and `false` otherwise.",
+        "schema": {
+            "type": "boolean"
+        }
+    },
+    "examples": [
+        {
+            "arguments": {
+                "x": 1
+            },
+            "returns": false
+        },
+        {
+            "arguments": {
+                "x": "Test"
+            },
+            "returns": false
+        },
+        {
+            "arguments": {
+                "x": null
+            },
+            "returns": false
+        }
+    ],

```

Comment:
there is no example where `true` is returned now, but as far as I understand the structure of the examples, it's technically not possible to construct such an example (even with the existence of the `nan` process to produce a NaN value)

Reply:
Indeed. I may remove these examples and add them to the tests (see #468) when we merged those.